javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,20 @@
+const fullName = 'Julie Luangpraxay';
+const isCool = true;
+const totalPets = 1;
+
+console.log('value of fullName:', fullName);
+console.log('value of isCool:', isCool);
+console.log('value of totalPets:', totalPets);
+
+console.log('typeof fullName:', typeof fullName);
+console.log('typeof isCool', typeof isCool);
+console.log('typeof totalPets:', typeof totalPets);
+
+const unknown = null;
+
+console.log('value of unknown', unknown);
+console.log('typeof unknown:', typeof unknown);
+
+const never = undefined;
```

Comment:
This should be using the `let` keyword, and you shouldn't be assigning it a value. It should just be `let never`. Since we don't assign it a value, it will automatically be undefined.

Reply:
Fixed, thank you!

<img width="585" alt="Screenshot 2023-07-10 at 9 43 46 PM" src="https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/42b837e2-012b-41f0-a3b8-534b48957827">